internal/command/console/console.go

```diff
@@ -159,7 +224,37 @@ func runConsole(ctx context.Context) error {
 		return err
 	}
 
-	return ssh.Console(ctx, sshClient, appConfig.ConsoleCommand, true)
+	var consoleCommand string
+	if flag.IsSpecified(ctx, "entrypoint") {
+		consoleCommand = flag.GetString(ctx, "entrypoint") + " " + strings.Join(flag.Args(ctx), " ")
+	} else {
+		if flag.IsSpecified(ctx, "image") {
+			resp, err := gql.GetImageConfig(ctx, apiClient.GenqClient, appName, flag.GetString(ctx, "image"))
+			if err != nil {
+				return fmt.Errorf("failed to get image config: %w", err)
+			}
+			type config struct {
+				Config struct {
+					Entrypoint []string
+					WorkingDir string
+				}
+			}
+			var cfg config
+			err = mapstructure.Decode(resp.App.Image.Config, &cfg)
+			if err != nil {
+				return fmt.Errorf("failed to decode config: %w", err)
+			}
+			consoleCommand = fmt.Sprintf(
+				"/bin/sh -c 'cd %s && %s %s'",
+				cfg.Config.WorkingDir,
+				strings.Join(cfg.Config.Entrypoint, " "),
+				strings.Join(flag.Args(ctx), " "),
```

Comment:
It seems unnecessary fragile because of https://owasp.org/www-community/attacks/Command_Injection.

Sadly https://pkg.go.dev/golang.org/x/crypto/ssh doesn't support shell-less execution like os/exec. Would https://github.com/kballard/go-shellquote work?